Code/ServerApp/Source/NetImguiServer_App.h

```diff
@@ -6,9 +6,30 @@
 //=============================================================================================
 // SELECT RENDERING/OS API HERE
 //=============================================================================================
-#define HAL_API_PLATFORM_WIN32_DX11		1
-#define HAL_API_PLATFORM_GLFW_GL3		0														// Currently only compiles in release (library include compatibility)
-#define HAL_API_PLATFORM_SOKOL			0
+// 1. Either edit the lines below to select the desired backend
+#if !defined(NETIMGUI_SERVER_APP_BACKEND_SOKOL) && !defined(NETIMGUI_SERVER_APP_BACKEND_GLFW_GL3) && !defined(NETIMGUI_SERVER_APP_BACKEND_WIN32_DX11)
```

Comment:
I think that instead of having to list every case here, it should be a series of ifdef / else until the default behaviour is kept in the final else, without having to rely on defines.

```
#if defined(xxx)
// xxx
#elif defined(yyy)
// yyy
#elif ....
#else
// Default values here
#endif
```

Reply:
Ok, the code now reads 

```cpp
// Select the backend from with cmake (with -DNETIMGUI_SERVER_APP_BACKEND_WIN32_DX11, etc)
// Note : HAL_API_PLATFORM_GLFW_GL3 currently only compiles in release (library include compatibility)
#if defined(NETIMGUI_SERVER_APP_BACKEND_WIN32_DX11)
    #define HAL_API_PLATFORM_WIN32_DX11	1
    #define HAL_API_PLATFORM_SOKOL		0
    #define HAL_API_PLATFORM_GLFW_GL3	0
#elif defined(NETIMGUI_SERVER_APP_BACKEND_GLFW_GL3)
    #define HAL_API_PLATFORM_WIN32_DX11	0
    #define HAL_API_PLATFORM_SOKOL		0
    #define HAL_API_PLATFORM_GLFW_GL3	1
#elif(defined(NETIMGUI_SERVER_APP_BACKEND_SOKOL))
    #define HAL_API_PLATFORM_WIN32_DX11	0
    #define HAL_API_PLATFORM_SOKOL		1
    #define HAL_API_PLATFORM_GLFW_GL3	0
#else
    // Or manually edit the lines below to select the desired backend
    #define HAL_API_PLATFORM_WIN32_DX11		1
    #define HAL_API_PLATFORM_GLFW_GL3		0
    #define HAL_API_PLATFORM_SOKOL			0
#endif
```